### PR TITLE
Issue/fix akka runner

### PR DESCRIPTION
**Pull request by c3riley**
akka runner from cli had a number of issues, with threadlocal variables bleeding thru and no aggregate reporting. Fix the issues and added simple test that will be expanded.  Added very simple summary html report to start.

### CODE REVIEW

File: testah-junit/src/main/java/org/testah/framework/testPlan/AbstractTestPlan.java [hunk 1]
```diff
@@ -33,22 +31,26 @@
...
+
 /**
  * The Class AbstractTestPlan.
  */
 @ContextHierarchy({ @ContextConfiguration(classes = TestConfiguration.class) })
 public abstract class AbstractTestPlan extends AbstractJUnit4SpringContextTests {
 
     /** The test plan. */
-    private static ThreadLocal<TestPlanDto> testPlan;
+    private static ThreadLocal<TestPlanDto> testPlan = new ThreadLocal<>();;
```
Comment (gweiske) [inline testah-junit/src/main/java/org/testah/framework/testPlan/AbstractTestPlan.java]: duplicate ;; at end of line, TestCaseDto and TestStepDto, in the next few lines as well.

File: testah-junit/src/main/java/org/testah/runner/TestahJUnitRunner.java [hunk 2]
```diff
@@ -1,76 +1,90 @@
...
+
+    /**
+     * Run tests.
+     *
+     * @param numConcurrent        the num concurrent
+     * @param jUnitTestPlanClasses the j unit test plan classes
+     * @return the list
+     */
+    public List<ResultDto> runTests(final int numConcurrent, final Set<Class<?>> jUnitTestPlanClasses) {
+        try {
```
Comment (gweiske) [inline testah-junit/src/main/java/org/testah/runner/TestahJUnitRunner.java]: public List<ResultDto> runTests(final int numConcurrent, final List<Class<?>> jUnitTestPlanClasses) seems to be a wrapper for this method. However, the wrapper also has some logic to deal with list of test plans being null and it has
setInUse(true);
setInUse(false);
Maybe move that logic into this class?

File: testah-junit/src/main/java/org/testah/driver/http/AbstractHttpWrapper.java [hunk 3]
```diff
@@ -72,7 +70,7 @@
     private int defaultPoolSize = 100;
 
     /** The Default max per route. */
-    private final int DefaultMaxPerRoute = 100;
+    private int DefaultMaxPerRoute = 100;
```
Comment (gweiske) [inline testah-junit/src/main/java/org/testah/driver/http/AbstractHttpWrapper.java]: Not final? I.e. can be modified?

Comment (gweiske) [inline testah-junit/src/main/java/org/testah/driver/http/AbstractHttpWrapper.java]: no setter for DefaultMaxPerRoute - keep it final?

**Comment by gweiske** [comment]
👍
